### PR TITLE
type hinting and code formatting

### PR DESCRIPTION
**Pull request by TomK**
I noticed a few issues with the code as I was formatting, however they are outside the scope of this PR.  I will raise some issues as they will require executable code to change.

Please note i did not resolve the param/return of setAddress/getAddress in `PPOpenIdUserinfo` to an import as the full class name is required for `PPModel::fromArray`.   Imports could be dereferenced using a helper class like `TokenParser` from https://github.com/doctrine/annotations.  However if this these classic apis are essentially deprecated, theres not much point in spending the time on it.

### CODE REVIEW

File: lib/PayPal/Auth/IPPThirdPartyAuthorization.php [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 namespace PayPal\Auth;
+
 /**
- * 
+ *
  * Marker interface for implementations of third party authorization.
- * Third-party authorization allows you to make an API call on some 
- * other account�s behalf 
+ * Third-party authorization allows you to make an API call on some
+ * other account�s behalf
```
Comment (jaypatel512) [inline lib/PayPal/Auth/IPPThirdPartyAuthorization.php]: Not sure if this is valid character here.

File: lib/PayPal/Auth/Oauth/AuthSignature.php [hunk 2]
```diff
@@ -1,37 +1,42 @@
 <?php
 namespace PayPal\Auth\Oauth;
-//PayPal specific modification
+
+    //PayPal specific modification
```
Comment (jaypatel512) [inline lib/PayPal/Auth/Oauth/AuthSignature.php]: The identation seems to be wrong here.

**Comment by TomK** [comment]
thanks, i'll take a look

**Comment by jaypatel512** [comment]
Hey @TomK !

I think there is a tiny formatting issue with the space/tabs in all files. Just trying to reproduce this in my PHP Storm with PSR1/PSR2 settings renders different output. can you please verify your settings to be the default PSR1/2 ?

Thanks again for your wonderful help @TomK !

**Comment by TomK** [comment]
@jaypatel512 
I have also re-run auto formatting across `/lib` with the default PSR1/PSR2 code styles. There wasn't many changes however.  Only what can be seen in 570555e.
I have also manually corrected the character encoding issue in `IPPThirdPartyAuthorization`.

**Comment by TomK** [comment]
Closing while I review breaking changes.

**Pull request closed without merging**